### PR TITLE
new function "paradox-menu-copy-homepage-as-kill"

### PR DESCRIPTION
**Pull request by Dickby**

### CODE REVIEW

File: paradox-menu.el [hunk 1]
```diff
@@ -789,6 +790,19 @@ PKG is a symbol.  Interactively it is the package under point."
                (propertize (symbol-name pkg)
                            'face 'font-lock-keyword-face)))))
 
+(defun paradox-menu-copy-homepage-as-kill (pkg)
+  "Save the homepage of package named PKG as kill
```
Comment (Malabarba) [inline paradox-menu.el]: Missing `.` at the end of the sentence.

File: paradox-menu.el [hunk 2]
```diff
@@ -789,6 +790,19 @@ PKG is a symbol.  Interactively it is the package under point."
...
+PKG is a symbol.  Interactively it is the package under point."
+  (interactive '(nil))
+  (let ((url (paradox--package-homepage
+	      (paradox--get-or-return-package pkg))))
```
Comment (Malabarba) [inline paradox-menu.el]: Missing 1 space of indentation.

**Comment by Dickby** [comment]
I'm not that familiar to (ma)git. The reason why it lookes like this now is because i tried to sqash all changes together.

**Comment by Malabarba** [comment]
No problem. Thanks for the PR!

**Pull request merged**